docs/why.md

```diff
@@ -1,38 +1,146 @@
-## Django's Atomic
-Django's `atomic` ensures database changes are committed together-or-not-at-all. It creates a savepoint or a transaction depending on two factors:
+# Django's Atomic
 
-- The arguments passed to it (`durable` and `savepoint`).
+This doc will discuss the behaviours available through Django's `atomic`,
+and the outcomes people are usually trying to achieve with it.
+It goes on to outline some pitfalls that can result from using `atomic`,
+and how Subatomic avoids them.
+
+Django's `atomic` ensures database changes are committed together-or-not-at-all.
+It creates a savepoint or a transaction depending on two factors:
+
+- The arguments passed to it (`durable=` and `savepoint=`).
 - If a database transaction is already open.
 
-Specifically, the **Behaviours** which `atomic` exhibits are:
+## Behaviours
+
+The **Behaviours** which `atomic` exhibits are:
```

Comment:
We're bolding 'Behaviours' here, but then italicize it whenever we refer to it. Same with 'Outcomes' -- should we be consistent?